tools/RunGen.cpp

```diff
@@ -680,6 +801,10 @@ Usage: $NAME$ argument=value [argument=value... ] [flags]
         Override the default number of benchmarking iterations; ignored if 
         --benchmark is not also specified.
 
+    --benchmark_warmup=NUM: 
```

Comment:
Yeah, probably overthinking it: I put it in to experiment with something and didn't take it back out.